calendar/calendar.yaml

```
@@ -98,7 +98,7 @@
   organizer: juliusvonkohout
 
 - id: kf037
-  name: Kubeflow Notebooks WG Meeting (US + EMEA)
+  name: Kubeflow Notebooks 2.0 WG Meeting (US + EMEA)
```

@ederign I think it's more clear if we add `+` in the middle, so it shows we are doing both the Notebooks 2.0 and WG meeting. 

```suggestion
  name: Kubeflow Notebooks 2.0 + WG Meeting (US + EMEA)
```